validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache

```diff
@@ -1,22 +1,32 @@
+-
+  metricName: ecs.task.memory.reserved_{{testingId}}
+  namespace: {{metricNamespace}}
+  dimensions:
```

Comment:
let's wait until you have the real dimensions, then we can merge

Reply:
We are planning release with empty dimensions. Will create a new PR once the label/dimensions settings feature is available.